include/uapi/sound/sof/ext_manifest.h

```diff
@@ -51,11 +51,24 @@ struct sof_ext_man_header {
 
 /* Now define extended manifest elements */
 
+/* Extended manifest elements identificators */
```

Comment:
element identifiers? or just element types maybe? Im not sure identificator is a proper word

Reply:
changed